TTHAnalysis/python/plotter/mcPlots.py

```diff
@@ -990,7 +990,7 @@ def printOnePlot(self,mca,pspec,pmapIn,mytotal=None,makeCanvas=True,outputDir=No
                     signorms = doStackSignalNorm(pspec,pmap,options.showIndivSigShapes or options.showIndivSigs,extrascale=options.signalPlotScale, norm=not options.showIndivSigs)
                     for signorm in signorms:
                         if outputDir: 
-                            signorm.SetDirectory(outputDir); outputDir.WriteTObject(signorm)
+                            signorm.SetDirectory(outputDir); #outputDir.WriteObjectAny(signorm)
```

Comment:
Do we need this comment after all? If yes, we should also notify Marco.

Reply:
We indeed need to comment out this: `outputDir.WriteTObject(signorm)`. Could you please revert to the commented out version? I will merge the PR after that. Thanks!